code/modules/mob/living/death.dm

```diff
@@ -111,8 +111,8 @@
 	if(stat == DEAD)
 		return FALSE
 
-	if(!gibbed && (death_sound || death_message || (living_flags & ALWAYS_DEATHGASP)))
-		INVOKE_ASYNC(src, TYPE_PROC_REF(/mob, emote), "deathgasp")
+/* 	if(!gibbed && (death_sound || death_message || (living_flags & ALWAYS_DEATHGASP)))
+		INVOKE_ASYNC(src, TYPE_PROC_REF(/mob, emote), "deathgasp") */
```

Comment:
The lack of deathgasps will make it extremely difficult to discern whether the person was knocked into crit or actually killed by someone, making murder not just a blurry mess ingame, but also for ooc stuff like tickets.

Reply:
intentional, I want to blur what is killing and what is critical, I want people declared dead by medical persons